enhancements/console/customize-perspectives.md

```diff
@@ -0,0 +1,294 @@
+---
+title: customize-perspectives
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-07-26
+last-updated: 2022-08-25
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Perspectives 
+
+## Summary
+
+OpenShift Container Platform (OCP) web console provides two perspectives: Administrator and Developer. 
+The Developer perspective provides workflows specific to developer use cases 
+like create, deploy and monitor applications, while Administrator perspective 
+is responsible for managing the cluster resources, users, and projects.
+
+This document provides details about how cluster administrator 
+can hide user perspectives. 
+
+![Perspectives switch dropdown](perspectives-switch.png)
+
+## Motivation
+
+Cluster administrator wants to hide the "Administrator" perspective 
+from non-privileged and to hide the "Developer" 
+perspective for all users.
+
+### User Stories
+
+#### Story 1
+
+As a cluster admin, I want to hide "Administrator" perspective 
+from all non-privileged users so that non-privileged users
+cannot access "Administrator" perspective for managing the cluster resources, 
+users, and projects. Non-privileged users are users that cannot get
+all `namespaces` (based on RBAC check)
+
+#### Story 2
+
+As a cluster admin, I want to hide "Developer" perspective
+for all users.
+
+#### Story 3
+
+As a cluster admin, I want to hide any perspective contributed by a plugin.
```

Comment:
Because the plugin may not be relevant to certain users? Again, tell me why you want to do this

Reply:
A plugin can add multiple features to the console, the users are may be interested 
in other features of the plugin but not the perspective.

It might be also interesting for a customer to show this new perspective
only to users with specific permissions.

Reply:
Useful context, please add to this user story